notification/twilio/sms.go

```diff
@@ -266,7 +266,10 @@ func (s *SMS) ServeMessage(w http.ResponseWriter, req *http.Request) {
 	}
 
 	body := req.FormValue("Body")
-	if strings.Contains(strings.ToLower(body), "stop") {
+	// twilio unsubscribes on single-word matches
+	// i.e. "please stop" will not result in unsubscribing on their end.
+	checkBody := func(str string) bool { return strings.ToLower(body) == str }
+	if checkBody("stop") || checkBody("stopall") || checkBody("unsubscribe") || checkBody("cancel") || checkBody("end") || checkBody("quit") {
```

Comment:
Maybe we can just break this out into it's own func -- should make things easier when we add support for the start codes as well. We can also get away with calling `ToLower` once for all options and avoid all the `||` conditions.

```go
func isStopMessage(body string) bool {
  switch strings.ToLower(body) {
  case "stop", "stopall",  "...":
    return true
  }

  return false
}

func ServeMessage() {
  // ...
  body := req.FormValue("Body")
  if isStopMessage(body) {
    // ...
  }
  // ...
}
```